### PR TITLE
Dalys

### PR DESCRIPTION
**Pull request by lmhaile**
Resolves comments in [this PR](https://github.com/mrc-ide/postie/pull/2).

Additionally incorporates updated treatment scaler per Nora's research.

### CODE REVIEW

File: R/epi.R [hunk 1]
```diff
@@ -54,8 +58,8 @@ dalys <- function(
     mild_disability_weight = 0.006,
     moderate_disability_weight = 0.051,
     severe_disability_weight = 0.133,
-    clinical_episode_length = 0.01375,
-    severe_episode_length = 0.04795
+    clinical_episode_length = 0.01375 * 365,
```
Comment (pwinskill) [inline R/epi.R]: These should not be multiplied by 365. The output of the DALYs is in years and the units of these parameters are in years already. 
E.g. The length of a clinical episode is ~ 5 days = 0.01375 years, so the number of diability adjusted life _years_ is cases * weight * 0.1375 years.

File: R/rates.R [hunk 2]
```diff
@@ -39,13 +47,13 @@ get_rates <- function(x,
...
                       scaler = 0.215,
-                      treatment_scaler = 0.5,
+                      treatment_scaler = 0.42,
                       aggregate_age = FALSE,
                       mild_disability_weight = 0.006,
                       moderate_disability_weight = 0.051,
                       severe_disability_weight = 0.133,
-                      clinical_episode_length = 0.01375,
-                      severe_episode_length = 0.04795){
+                      clinical_episode_length = 0.01375 * 365,
```
Comment (pwinskill) [inline R/rates.R]: See other comment

File: tests/testthat/test-epi.R [hunk 3]
```diff
@@ -24,8 +24,8 @@ test_that("treatment scaling works", {
   output <- treatment_scaling(mock_data, treatment_scaler = 0)
   expect_equal(output$severe, mock_data$severe)
 
-  output_ts <- treatment_scaling(mock_data, treatment_scaler = 0.5, baseline_treatment = 0.4)
-  expect_equal(output_ts$severe, mock_data$severe * ((0.5 * mock_data$ft + (1 - mock_data$ft)) / (0.5 * 0.4 + (1 - 0.4))))
+  output_ts <- treatment_scaling(mock_data, treatment_scaler = 0.43, baseline_treatment = 0.4)
+  expect_equal(output_ts$severe, mock_data$severe * ((0.57 * mock_data$ft + (1 - mock_data$ft)) / (0.57 * 0.4 + (1 - 0.4))))
```
Comment (pwinskill) [inline tests/testthat/test-epi.R]: Can we change `0.57` to `(1 - 0.43)` in these examples, just so it is obvious where it came from?

File: tests/testthat/test-rates.R [hunk 4]
```diff
@@ -24,9 +24,9 @@ test_that("get rates work", {
     clinical = 1:2 / (1:2 * 10),
     severe = (1:2 / 10) / (1:2 * 10),
     mortality = (1:2 / 10) / (1:2 * 10),
-    yld_pp = 0.0001338985,
+    yld_pp = 0.0001338985*365,
```
Comment (pwinskill) [inline tests/testthat/test-rates.R]: As before

**Review by pwinskill — CHANGES_REQUESTED** [review]

**Comment by lmhaile** [comment]
Comments incorporated now!

**Comment by pwinskill** [comment]
Are there still some * 365 in get_rates?

**Comment by lmhaile** [comment]
> Are there still some * 365 in get_rates?

You're correct, should be fixed with the latest commit!

**Pull request merged**